src/App.jsx

```diff
@@ -1,40 +1,82 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selected, setSelected] = useState('tab-1');
+  const [title, setTitle] = useState('');
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+  return (
+    <div className="section">
+      <h1 className="title">
+        Selected tab is
+        {title}
+      </h1>
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            <li
```

Comment:
map a list of `<li>` and then you can use values from an object to pass into right places 
like 
```
              <a
                href={`#${tab.id}`}
                data-cy="TabLink"
                onClick={() => {
                  setSelected(tab);
                }}
              >
                {tab.title}
              </a>
```